apps/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -77,6 +78,10 @@ export abstract class BaseInstallAction extends BaseRushAction {
       description: `Overrides the default maximum number of install attempts.`,
       defaultValue: RushConstants.defaultMaxInstallAttempts
     });
+    this._ignoreHooksParameter = this.defineFlagParameter({
+      parameterLongName: '--ignore-hooks',
+      description: `Overrides execution of "preInstall" and "postInstall" event hooks. Meant for use on CI machines.`
```

Comment:
> Meant for use on CI machines.

Not sure if this bit is necessary. It's true in our case, but not necessarily for others

Reply:
Updated.